release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -196,6 +196,18 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-operators"]
+=== Operators
+
+[id="ocp-4-6-cloud-credential-operator-mode-support"]
+==== Support for Cloud Credential Operator modes
+
+The xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator (CCO)]
+can now be configured to operate in the following modes: `Mint`, `Passthrough`,
+and `Manual`. This feature provides transparency and flexibility in how the CCO
+uses cloud credentials to process `CredentialRequests` in the cluster for
+installation and other tasks.
```

Comment:
New feature summary for [CO-967: CCO: explicit mode support](https://issues.redhat.com/browse/CO-967)
(Note that updated CCO content from that link is in a different branch)

Reply:
Preview link: https://rns-4_6-ga-cco-features--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-cloud-credential-operator-mode-support